src/index.js

```diff
@@ -79,7 +79,8 @@ class Daemon extends EventEmitter {
     if (headless == null) {
       return cb(new Error('Could not load "headless" module'))
     }
-    headless((err, child, display) => {
+    var opts = {args: ['+extension', 'RANDR']}
```

Comment:
Those arguments won't work, see: https://github.com/kesla/node-headless/blob/master/index.js#L44. It only accepts display related options.